### ISSUE TITLE
npm-cli-login when called from the command line does not work with node 10.1.0

### ISSUE THREAD

**Issue by LivesleyWA** [postmanlabs/npm-cli-login#25]
Works with 8.11.0

```
http 201 https://registry.npmjs.org/-/user/org.couchdb.user:xxxxxx
fs.js:140
  throw new ERR_INVALID_CALLBACK();
  ^

TypeError [ERR_INVALID_CALLBACK]: Callback must be a function
    at maybeCallback (fs.js:140:9)
    at Object.fs.writeFile (fs.js:1258:14)
    at Object.writeFile (/usr/local/lib/node_modules/npm-cli-login/lib/login.js:120:12)
    at /usr/local/lib/node_modules/npm-cli-login/lib/index.js:22:25
    at ReadFileContext.callback (/usr/local/lib/node_modules/npm-cli-login/lib/login.js:69:20)
    at FSReqWrap.readFileAfterOpen [as oncomplete] (fs.js:334:13)

[Container] 2018/06/20 01:16:30 Command did not exit successfully npm-cli-login -u $NPM_USERNAME -p $NPM_PASSWORD -e $NPM_USER_EMAIL exit status 1
```

**Comment by SamuraiJack** [comment]
Same here..
```
nickolay@outpost:~$ npm-cli-login -u "samuraijack" -p "(ReplayYalper)" -e "nickolay8@gmail.com"
info attempt registry request try #1 at 16:00:53
http request PUT https://registry.npmjs.org/-/user/org.couchdb.user:samuraijack
http 201 https://registry.npmjs.org/-/user/org.couchdb.user:samuraijack
fs.js:140
  throw new ERR_INVALID_CALLBACK();
  ^

TypeError [ERR_INVALID_CALLBACK]: Callback must be a function
    at maybeCallback (fs.js:140:9)
    at Object.fs.writeFile (fs.js:1258:14)
    at Object.writeFile (/usr/lib/node_modules/npm-cli-login/lib/login.js:120:12)
    at /usr/lib/node_modules/npm-cli-login/lib/index.js:22:25
    at /usr/lib/node_modules/npm-cli-login/lib/login.js:69:20
    at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:422:3)
```

**Comment by wooderz** [comment]
This has been open since June.. any chance of a fix as it's stopping our build servers progressing now

**Comment by danthegoodman** [comment]
For others having this issue, I found `npm-cli-adduser` as an alternative command.

**Comment by czardoz** [comment]
Fixed in v0.1.1 (it's published now)
